guides/Getting Started/WebEngine Support/index.md

```diff
@@ -0,0 +1,123 @@
+With the introduction of WebEngine applications, the policy server requires additional setup in order to be able to support them. This is because WebEngine apps need to be uploaded and servable for execution by supported HMIs, and the policy server provides the information necessary for downloading these app bundles. The implementation details are up to the policy server user, as the method of hosting these WebEngine app bundles may depend on the environment and manner in which the policy server is running. 
```

Comment:
```suggestion
# WebEngine Support

With the introduction of WebEngine applications, the policy server requires additional setup in order to be able to support them. This is because WebEngine apps need to be uploaded and servable for execution by supported HMIs, and the policy server provides the information necessary for downloading these app bundles. The implementation details are up to the policy server user, as the method of hosting these WebEngine app bundles may depend on the environment and manner in which the policy server is running. 
```